api/source/service/STIGService.js

```diff
@@ -907,7 +907,7 @@ exports.deleteRevisionByString = async function(benchmarkId, revisionStr, svcSta
 
   let connection;
   try {
-    let [input, version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
+    let [version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
```

Comment:
regex returns what it captures positionally, so we need "input" here, even though it is not used.